src/build.d

```diff
@@ -456,65 +437,53 @@ alias zip = makeDep!((builder, dep) => builder
     })
 );
 
-alias html = memoize!(function()
-{
-    const haveDmd = env.get("DMD", null).length > 0;
+alias html = makeDep!((htmlBuilder, htmlDep) {
+    htmlBuilder
+        .name("html")
+        .description("Generate html docs, requires DMD and STDDOC to be set");
+    if (env.get("DMD", null).length == 0)
```

Comment:
I know that you're simply refactoring the code, but I think it's better if you delete this `if` statement and add `dmdExe` to the dependencies and use that to produce HTML.

Reply:
The Cybershadow DAutoTest uses the DMD environment variable to select which compile to generate the docs with.  I have a feeling that changing it will break DAutoTest, however, I will try in a future commit to see if using `dmdExe` works.